pyClient/zeth/joinsplit.py

```diff
@@ -601,46 +513,55 @@ def _compute_proof_hashes(proof_json: GenericProof) -> Tuple[bytes, bytes]:
         if key != "inputs":
             proof_elements.extend(proof_json[key])
     return (
-        sha256(encode_message_to_bytes(proof_elements)).digest(),
-        sha256(encode_pub_input_to_hash(proof_json["inputs"])).digest())
+        encode_message_to_bytes(proof_elements),
+        encode_message_to_bytes(proof_json["inputs"]))
 
 
 def sign_mix_tx(
+        signing_keypair: SigningKeyPair,  # Ephemeral signing key, tied to proof
         sender_eph_pk: EncryptionPublicKey,  # Ephemeral key used for encryption
         ciphertexts: List[bytes],  # Encyrpted output notes
         proof_json: GenericProof,  # Proof for the mix transaction
-        signing_keypair: SigningKeyPair  # Ephemeral signing key, tied to proof
 ) -> int:
+    """
+    Generate a Schnorr signature on the hash of the ciphertexts, proofs and
```

Comment:
`Schnorr signature` -> should be abstracted away from this file (ie: Schnorr should not be mentioned)

`joinsplit_sign` signs the joinsplit description with the `JoinSplitSigPrivKey`, by using the imported signing library. Let's keep the actual algo used and this function decoupled as much as possible

Reply:
Removed the reference to Schnorr